staging/2021/benchmarks/benchmarks.csv

```diff
@@ -1,6 +1,5 @@
 Table 2: Historical MIPS Quality Measure Benchmark Results; created using PY2018 data and PY2020 Eligibility Rules,,,,,,,,,,,,,,,,,
 Measure_Name,Measure_ID,Collection_Type,Measure_Type,Benchmark,Standard_Deviation,Average,Decile_3,Decile_4,Decile_5,Decile_6,Decile_7,Decile_8,Decile_9,Decile_10,Topped_Out,Seven_Point_Cap,High_Priority
-Diabetes: Hemoglobin A1c (HbA1c) Poor Control (>9%),1,Medicare Part B Claims,Intermediate Outcome,Y,,15.6727392550143,79.99 - 70.01,70 - 60.01,60 - 50.01,50 - 40.01,40 - 30.01,30 - 20.01,20 - 10.01,<= 10,No,No,Y
```

Comment:
You'll keep these rows, but mark "N" for the `Benchmark` and replace the `Average` and `Decile_X` columns with `--`. The measures are still valid, but not the benchmarks. Will need to do the same for 117 below.